mlir/lib/Conversion/MathToLLVM/MathToLLVM.cpp

```diff
@@ -307,6 +341,8 @@ void mlir::populateMathToLLVMConversionPatterns(
     bool approximateLog1p, PatternBenefit benefit) {
   if (approximateLog1p)
     patterns.add<Log1pOpLowering>(converter, benefit);
+  patterns.add<IsNaNOpLowering, IsFiniteOpLowering>(converter);
+
   // clang-format off
   patterns.add<
     AbsFOpLowering,
```

Comment:
```suggestion
  // clang-format off
  patterns.add<
    IsNaNOpLowering,
    IsFiniteOpLowering,
    AbsFOpLowering,
```

Is there a reason for those not to be here?